OrbitCore/TracepointEventBuffer.h

```diff
@@ -37,8 +37,12 @@ class TracepointEventBuffer {
     }
   }
 
+  uint32_t GetNumTracepointsForThreadId(int32_t thread_id) const;
+
  private:
   int32_t kNotTargetProcessThreadId = -2;
+  int32_t no_total_tracepoints_ = 0;
+  int32_t no_tracepoints_not_in_target_process_ = 0;
```

Comment:
Same here: `num_tracepoints_not_in_target_process_` or `tracepoints_not_in_target_process_count_`.

Reply:
Of course! Thanks you!